e2e/node_package/pnpm-lock.yaml

```diff
@@ -0,0 +1,76 @@
+lockfileVersion: 5.3
```

Comment:
could you add a `.gitattributes` file so GH stops asking us to review these generated files?